### PR TITLE
Changed MODULE_VERSION in gdrdrv to follow GDRDRV_VERSION

### PR DESCRIPTION
**Pull request by pakmarkthub**
Problem:
- Issue #82.

This PR:
- modifies MODULE_VERSION in gdrdrv from hard-code to GDRDRV_*_VERSION.
- the idea about stringify preprocessor values comes from https://stackoverflow.com/questions/240353/convert-a-preprocessor-token-to-a-string.

### CODE REVIEW

**Comment by drossetti** [comment]
Love it, thanks

**Comment by drossetti** [comment]
actually STRINGIFY might be common, so I suggest to protect it with  #if !defined

**Comment by pakmarkthub** [comment]
Decided to add _GDRDRV_ prefix to those two preprocessor functions. This is because we cannot guarantee that other _STRINGIFY would have the same syntax as we expected.

**Comment by pakmarkthub** [comment]
@drossetti Please re-review the PR.

File: src/gdrdrv/gdrdrv.h [hunk 1]
```diff
@@ -23,9 +23,13 @@
 #ifndef __GDR_DRV_H__
 #define __GDR_DRV_H__
 
+#define _GDRDRV_STRINGIFY(s)           #s
+#define _GDRDRV_TOSTRING(s)            _GDRDRV_STRINGIFY(s)
```
Comment (drossetti) [inline src/gdrdrv/gdrdrv.h]: usually identifiers starting with underscore are considered "reserved".
can you get rif of it?

**Comment by pakmarkthub** [comment]
@drossetti I have removed the underscore.

**Pull request merged**